base-tppi2/scripts/rftools.zs

```diff
@@ -324,3 +320,19 @@ recipes.addShaped(<rftools:storageModuleItem:2>, [
   [blockQuartz, annihilationCore, blockQuartz],
   [formationCore, <rftools:storageModuleItem:1>, formationCore],
   [glow, null, glow]]);
+
+recipes.remove(<rftools:remoteStorageBlock>);
+recipes.addShaped(<rftools:remoteStorageBlock>, [
+  [quartz,<Mekanism:TeleportationCore>,quartz],
+  [<rftools:modularStorageBlock>,rftmachineframe,<rftools:modularStorageBlock>],
+  [iron,<rftools:modularStorageBlock>,iron]]);
+
+recipes.remove(<rftools:environmentalControllerBlock>);
+recipes.addShaped(<rftools:environmentalControllerBlock>,[
+  [beacon,quartz,beacon],
+  [quartz,rftmachineframe,quartz],
+  [beacon,quartz,beacon]]);
+
+recipes.remove(<rftools:dimensionalShardItem>);
+recipes.addShapeless(<rftools:dimensionalShardItem>*16,
```

Comment:
I wonder if it wouldn't be more thematic to use mana. But I'll merge this as-is, and you can send a separate PR if you want.

Reply:
i could probably have them use the transmutation pool...

Thing is i didn't want to gate any of RFtools behind Botania, and honestly the things are so common in dims as is, that even using ender pearls seems a little expensive...